app/helpers/sessions_helper.rb

```diff
@@ -17,4 +17,10 @@ def logout
 		flash[:danger] = "Successfully Logged out!"
 	end
 
+	def admin_user
+		User.find_by(id: session[:user_id])
+	end
```

Comment:
Please remove admin_user method. You are not using this method in your application